site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -13,15 +13,20 @@ export const Language = {
   suspendDialogTitle: "Suspend user",
   suspendDialogAction: "Suspend",
   suspendDialogMessagePrefix: "Do you want to suspend the user",
+  activateDialogTitle: "Activate user",
+  activateDialogAction: "Activate",
+  activateDialogMessagePrefix: "Do you want to activate the user",
 }
 
 export const UsersPage: React.FC = () => {
   const xServices = useContext(XServiceContext)
   const [usersState, usersSend] = useActor(xServices.usersXService)
   const [rolesState, rolesSend] = useActor(xServices.siteRolesXService)
-  const { users, getUsersError, userIdToSuspend, userIdToResetPassword, newUserPassword } = usersState.context
+  const { users, getUsersError, userIdToSuspend, userIdToActivate, userIdToResetPassword, newUserPassword } =
+    usersState.context
```

Comment:
Wondering what the ideal linting should be here. Something like:
```
const {
    users,
    getUsersError,
    userIdToSuspend,
    userIdToActivate, 
    userIdToResetPassword, 
    newUserPassword
} = usersState.context
```
Thoughts?

Reply:
Oh, sorry I missed this! I agree, that is nicer. I left it up to the formatter and it didn't print it as clearly. I can update on my next UI PR.

Reply:
I missed it in my initial review too. Next PR sounds good! I wonder if we can add something to the formatter though.

Reply:
Ah, yeah, just tried it out and we'd have to adjust the prettier rule. I think the [default is 80](https://prettier.io/docs/en/options.html) and [we've got it set to 120](https://github.com/coder/coder/blob/main/site/.prettierrc). This would be a good thing to chat about in FE variety if you'd like.